src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java

```diff
@@ -100,24 +93,10 @@ public void requestAndWriteKitRequests(List<LatestKitRequest> latestKitRequests)
                                                             String collaboratorParticipantId = KitRequestShipping.getCollaboratorParticipantId(latestKit.getBaseURL(), latestKit.getInstanceID(), latestKit.isMigrated(),
                                                                     latestKit.getCollaboratorIdPrefix(), (String) profile.get("guid"), (String) profile.get("hruid"), kitRequestSettings.getCollaboratorParticipantLengthOverwrite());
 
-                                                            if (kitHasSubKits) {
```

Comment:
A few cases of changing the `if (kitHasSubKits)... else {}` to preserve the `else` logic.